moabb/paradigms/p300.py

```diff
@@ -81,7 +82,12 @@ def used_events(self, dataset):
 
     def process_raw(self, raw, dataset):
         # find the events
-        events = mne.find_events(raw, shortest_event=0, verbose=False)
+        try:
+            events = mne.find_events(raw, shortest_event=0, verbose=False)
+        except ValueError:
```

Comment:
do we have to do this as a try/except, or can we test this in a better way?

Reply:
I could use `mne.utils._get_stim_channels(..., raise_error=False)` and check whether the result of that function is empty.

Reply:
I think that would be slightly clearer, since we'd know what the error is more explicitly